frontend/src/App.js

```diff
@@ -27,10 +30,9 @@ const App = () => {
   if (featuresLoaded) {
     return (
       <div>
-        <Heading appearance="H1">Epicenter</Heading>
-        <Text size="small" weight="normal">
-          The Learning Center
-        </Text>
+        <Provider store={store}>
+          <Rounting />
```

Comment:
I think we should put features loading logic inside redux, so that redux is ready as soon as features finish fetching. What do you think?